internal/ratiospoof/ratiospoof.go

```diff
@@ -0,0 +1,229 @@
+package ratiospoof
```

Comment:
This is looking much better! Great work

Reply:
Thank you for pointing out all this things ;)